src/main/java/github/weichware10/util/Tutorial.java

```diff
@@ -0,0 +1,90 @@
+package github.weichware10.util;
+
+import github.weichware10.util.Enums.ToolType;
+import github.weichware10.util.config.ConfigClient;
+
+
+/**
+ * Grundlegende Tutorial-Klasse.
+ *
+ * <p>Erbende Tutorial-Klassen müssen nur die Tutorial-Funktion implementieren.
+ *
+ * <p>Erbende Tutorial-Klassen müssen in ihrem Constructor super() callen.
+ *
+ * <p>Aufgerufen wird die start()-Funktion.
+ */
+public abstract class Tutorial {
+    private final ToolType toolType;
+    private final ConfigClient configClient;
+
+    /**
+     * Instaniziiert ein neues Tutorial-Objekt.
+     *
+     * @param configClient - der ConfigClient für die Einstellungen
+     * @param toolType - der ToolType des Tutorials
+     */
+    public Tutorial(ConfigClient configClient, ToolType toolType) {
+        this.configClient = configClient;
+        this.toolType = toolType;
+    }
+
+    /**
+     * Startet die Ausführung des Tutorials.
+     */
+    public void start() {
+        ;
+    }
+
+    /**
+     * findet heraus, ob für das Tutorial für das Tool angezeigt werden soll.
```

Comment:
Müsste es nicht so lauten:
```suggestion
     * findet heraus, ob für das Tool, dass Tutorial angezeigt werden soll.
```